apps/src/code-studio/districtDropdown/SchoolInfoBox.jsx

```diff
@@ -0,0 +1,145 @@
+        <option value="CA">California</option>
+        <option value="CO">Colorado</option>
+        <option value="CT">Connecticut</option>
+        <option value="DE">Delaware</option>
+        <option value="FL">Florida</option>
+        <option value="GA">Georgia</option>
+        <option value="HI">Hawaii</option>
+        <option value="ID">Idaho</option>
+        <option value="IL">Illinois</option>
+        <option value="IN">Indiana</option>
+        <option value="IA">Iowa</option>
+        <option value="KS">Kansas</option>
+        <option value="KY">Kentucky</option>
+        <option value="LA">Louisiana</option>
+        <option value="ME">Maine</option>
+        <option value="MD">Maryland</option>
+        <option value="MA">Massachusetts</option>
+        <option value="MI">Michigan</option>
+        <option value="MN">Minnesota</option>
+        <option value="MS">Mississippi</option>
+        <option value="MO">Missouri</option>
+        <option value="MT">Montana</option>
+        <option value="NE">Nebraska</option>
+        <option value="NV">Nevada</option>
+        <option value="NH">New Hampshire</option>
+        <option value="NJ">New Jersey</option>
+        <option value="NM">New Mexico</option>
+        <option value="NY">New York</option>
+        <option value="NC">North Carolina</option>
+        <option value="ND">North Dakota</option>
+        <option value="OH">Ohio</option>
+        <option value="OK">Oklahoma</option>
+        <option value="OR">Oregon</option>
+        <option value="PA">Pennsylvania</option>
+        <option value="RI">Rhode Island</option>
+        <option value="SC">South Carolina</option>
+        <option value="SD">South Dakota</option>
+        <option value="TN">Tennessee</option>
+        <option value="TX">Texas</option>
+        <option value="UT">Utah</option>
+        <option value="VT">Vermont</option>
+        <option value="VA">Virginia</option>
+        <option value="WA">Washington</option>
+        <option value="DC">Washington DC</option>
+        <option value="WV">West Virginia</option>
+        <option value="WI">Wisconsin</option>
+        <option value="WY">Wyoming</option>
+      </select>
+    </div>
+  );
+}
+
+SchoolState.propTypes = {
+  schoolState: React.PropTypes.string,
+  onChange: React.PropTypes.func
+};
+
+class SchoolInfoBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      country: "US",
+      schoolType: "",
+      schoolState: "",
+    };
+    this.handleCountryChange = this.handleCountryChange.bind(this);
+    this.handleSchoolTypeChange = this.handleSchoolTypeChange.bind(this);
+    this.handleSchoolStateChange = this.handleSchoolStateChange.bind(this);
+  }
+
+  handleCountryChange(e) {this.setState({country: e.target.value});}
+  handleSchoolTypeChange(e) {this.setState({schoolType: e.target.value});}
+  handleSchoolStateChange(e) {this.setState({schoolState: e.target.value});}
```

Comment:
I tried that initially, looks like we don't have experimental features enabled in Babel, though, so it failed. Will break up to multiple lines instead.